MuonAnalysis/MomentumScaleCalibration/plugins/MuScleFitMuonSelector.h

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
Technically the recommendation is to merge this header file with the cc file. Or if it's used by other stuff move it to the /interface/

Reply:
well, that's clearly outside the scope of this urgent PR.

Reply:
actually upon second look, this header is not a plugin in a cmssw sense, but a "standalone" class and it's just used locally for the plugins in this directory, so I think it's allowed to keep it like that.

Reply:
ok!